course-v2/layouts/partials/download_course_link_button.html

```diff
@@ -0,0 +1,10 @@
+{{- $trimmedBaseUrl := strings.TrimSuffix "/" site.BaseURL -}}
+{{- $downloadPageUrl := print $trimmedBaseUrl "/download" -}}
+{{- $downloadPageUrl = partial "page_url.html" (dict "context" .context "url" $downloadPageUrl) -}}
+<div class="mt-4">
```

Comment:
```suggestion
<div class="my-4">
```

Download button sticks to the bottom for pages having less content.  
<img width="257" alt="image" src="https://user-images.githubusercontent.com/93309234/200559517-d10ce615-3585-4a0f-bab6-0aae4cb1b61b.png">